//package com.indus.training.persist.impl;
//
//import java.nio.channels.AcceptPendingException;
//
//public class Sometest {
//
//	public static void main(String[] args) {
//		int a = 5;
//		int b = 6;
//		try {
//			System.out.println(a / 0);
//		} catch (Exception e) {
//			// System.out.println(e);
//		//	 throw new AcceptPendingException();
////			System.out.println("df");
//
//		}
//
//		System.out.println(b);
//
//	}
//
//}
package com.indus.training.persist.impl;

import java.io.IOException;

public class Sometest {

	public static void main(String[] args) throws ArithmeticException {
		int a = 5;
		int b = 6;
		System.out.println(a / 2);
		// throw new ArithmeticException();
		// System.out.println(b);

	}

}
